passenger-service/cab-aggreagator-passenger-service/passenger-service/src/main/java/com/modsen/passengerservice/dto/PassengerDto.java

```diff
@@ -0,0 +1,2 @@
+package com.modsen.passengerservice.dto;
+public record PassengerDto(Long id, String name, String email) {}
```

Comment:
Вот такой способ записи рекордов предпочтительный:
```suggestion
package com.modsen.passengerservice.dto;

public record PassengerDto(
    Long id,
    String name,
    String email
) {
}
```